### ISSUE TITLE
[Feature] Fine Tune the multi-node Etcd configurations.

### ISSUE THREAD

**Issue by ishan16696** [gardener/etcd-druid#265]
**Feature (What you would like to be added):**
The default settings in `multi-node etcd` works well  on a local network where the network latency is low. However, when using `multi-node etcd` deployed in multi zone where the average network latency can be high the `heartbeat interval` and `election timeout` settings may need tuning.

**Motivation (Why is this needed?):**
To avoid unnecessary leader-election from happening as well as to avoid sending too many heartbeats( if heartbeat interval is too low) as this will leads to increase the usage of CPU and network resources.

**Approach/Hint to the implement solution (optional):**
See more information regarding time parameters [here](https://etcd.io/docs/v3.4/tuning/#time-parameters)

**Comment by ishan16696** [comment]
@shreyas-s-rao  @unmarshall  Would it make sense to move this issue to `multi-node etcd zonal` Umbrella issue ?

**Comment by unmarshall** [comment]
> Would it make sense to move this issue to multi-node etcd zonal Umbrella issue ?

Yes, that would make sense. We will have to additionally do latency measurements when setting up etcd across nodes (in the same zone) and also across zones. An initial test was done by Shreyas but it was decided that this be done again. Recommendations will then have to be made and that will also be captured in the GEP.

**Comment by abdasgupta** [comment]
We need to observe the metrics mentioned in this issue in our usage to come up with the right values per cloud provider
